.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,222 @@
+id: 
```

Comment:
Which repo did you copy this file from?

Reply:
I took it from this PR: https://github.com/Azure/azure-functions-python-worker/pull/1285

Reply:
Based on the samples [here](https://eng.ms/docs/more/github-inside-microsoft/policies/resource-management), I don't think we need `id` or `owner`. Should I delete them?

Reply:
There's some inconsistencies in terms of the labels between the Python repo and what we use on Node.js repos. For example "needs-author feedback" vs "Needs: Author Feedback". I think the Node.js worker repo copied the labels from the host repo originally (example [here](https://github.com/Azure/azure-functions-host/blob/dev/.github/policies/resourceManagement.yml#L19)) and I stayed consistent with that when I created the library repo labels. I think the host repo is one of the more well-maintained repos for our team, so I kinda like staying consistent with them.

I'd start by copying the node worker [bot file](https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/.github/policies/resourceManagement.yml) instead of the python one. Unfortunately the bot doesn't work in that repo (example issue [here](https://github.com/Azure/azure-functions-nodejs-worker/issues/727) which I think the bot should've closed). So we'd need to figure that out, too

Reply:
I updated the file to follow the Node worker repo. I also reached out to the Github in Microsoft team to see if they can help us understand why the policy is not working.